src/main/java/io/debezium/connector/yugabytedb/YugabyteDBSchema.java

```diff
@@ -120,7 +120,7 @@ protected void refreshSchemaWithTabletId(TableId tableId,
                                              String schemaName,
                                              String tabletId) {
         String lookupKey = getLookupKey(tableId, tabletId);
-        if (cdcsdkSchemaPB == null) {
+        if (tabletIdToCdcsdkSchemaPB.get(lookupKey) == null || cdcsdkSchemaPB == null) {
```

Comment:
containsKey is better, can we have null value against the key?

Reply:
Addressed.